config/state.go

```diff
@@ -74,6 +77,16 @@ func (state *MixState) loadDefaultFormat() {
 	state.formatSource = "Mixer internal value"
 }
 
+func (state *MixState) loadDefaultPreviousMixVer() {
+	/* The LAST_VER is the default for PREVIOUS_MIX_VERSION */
+	lastVer, err := ioutil.ReadFile("update/image/LAST_VER")
```

Comment:
Typically, mixes are built in the current working directory, so relative paths will usually work. Since Mixin uses an alternative mix path (/usr/share/mix), a path relative to the current directory will likely be incorrect.

Also, there are existing cases on the master branch where state files use relative paths which will cause issues for Mixin.